config.yml

```diff
@@ -21,20 +21,18 @@
 opensuse/openSUSE-Tumbleweed-x86_64:
   salt:
     repos:
-      'oss': 'http://download.opensuse.org/tumbleweed/repo/oss/suse/'
+      'oss': 'http://download.opensuse.org/tumbleweed/repo/oss/'
       'up': 'http://download.opensuse.org/update/tumbleweed/'
       'ceph': 'https://download.opensuse.org/repositories/filesystems:/ceph/openSUSE_Tumbleweed/'
 #      'monitoring': 'https://download.opensuse.org/repositories/server:/monitoring/openSUSE_Tumbleweed/'
       'oA and monitoring': 'https://download.opensuse.org/repositories/filesystems:/openATTIC:/3.x/openSUSE_Tumbleweed/'
-      # workaround until salt-2017 is supported by deepsea
-      'salt-2016': 'http://download.opensuse.org/repositories/systemsmanagement:/saltstack:/products/openSUSE_Tumbleweed/'
+      'salt': 'http://download.opensuse.org/repositories/systemsmanagement:/saltstack:/products/openSUSE_Tumbleweed/'
     packages:
       all:
         - vim
-        - libcrypto42
-        - salt-minion-2016.11.04
+        - salt-minion
       admin:
-        - salt-master-2016.11.04
```

Comment:
should we still have salt-master on admin? maybe without version.

Reply:
Oh, yes, sorry did not push the change. Was a copy/paste error.